src/site/content/en/blog/preload-responsive-images/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Preloading responsive images
+subhead: Learn about new and exciting possibilities for preloading responsive images to ensure great user experience.
+authors:
+  - yoavweiss
+description: |
+  Learn about new and exciting possibilities for preloading responsive images to ensure great user experience.
+hero: hero.jpg
+alt: A wall with a bunch of images frames in different sizes.
+date: 2019-09-27
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+  - images
+---
+
+This article gives me an opportunity to discuss two of my favorite things: responsive images *and* preload. As someone who was heavily involved in developing both of those features, I'm super excited to see them working together!
+
+## Responsive images overview
+
+Suppose you're browsing the web on a screen that's 300 pixels wide, and the page just requested an image that's 1500 pixels wide. That page just wasted a lot of your cellular data because your screen can't do anything with all of that extra resolution. Ideally, the browser should fetch a version of the image that's just a *little* wider than your screen size, say 325 pixels. This ensures a high-resolution image without wasting data. And, even better, the image will load faster. [Responsive images](/serve-responsive-images/#serve-multiple-image-versions) enable browsers to fetch different image resources to different devices. You need to save multiple dimensions for each image and specify them in the `srcset` attribute. The `w` value tells the browser the width of each version. Depending on the device, the browser can choose the appropriate one:
+
+```html
+<img src="small.jpg"
+
+     srcset="small.jpg 500w,
+
+            medium.jpg 1000w,
+
+            large.jpg 1500w"
+
+            alt="…">
+```
```

Comment:
Please remove the extra spacing in this code

![image](https://user-images.githubusercontent.com/4713486/65640224-b2da7980-df9e-11e9-9695-bad8e932303e.png)